ocsci/pytest_customization/ocscilib.py

```diff
@@ -70,6 +82,32 @@ def pytest_configure(config):
     """
     if not config.getoption("--help"):
         process_cluster_cli_params(config)
+        # Add OCS related versions to the html report and remove extraneous metadata
+        if not ocsci_config.RUN['cli_params']['deploy']:
```

Comment:
If I understand it correctly, the following information gathering is skipped, if `--deploy` parameter is supplied? I'm not sure, if this is the best approach here, because since PR https://github.com/red-hat-storage/ocs-ci/pull/173 is merged, it is perfectly OK to run any test and use `--deploy` parameter to deploy the cluster as part of the test setup phase... I think, that in that case, it would make sense to gather those data before the test itself, but after the cluster is created.

Reply:
Handled --deploy case in commit 722e964